radlab-ui/webapp/src/components/forms/fields/ListField.tsx

```diff
@@ -49,8 +49,13 @@ const ListField: React.FC<ListField> = ({ variable }) => {
           return (
             <div>
               {values[variable.name].map((setvalue: string, index: number) => (
-                <div key={index} className="badge badge-info gap-2 m-1">
-                  <span>{setvalue}</span>
+                <div
+                  key={index}
```

Comment:
`key`s should NEVER be the index of the loop. Read more [here](https://robinpokorny.com/blog/index-as-a-key-is-an-anti-pattern/)

Reply:
Please use something unique from the **data** itself. In this case `variable.name` would probably work.